PRIVACY.md

```diff
@@ -0,0 +1,109 @@
+| [Semantic Scholar](https://www.semanticscholar.org/), powered by [Allen Institute for AI](https://allenai.org/) | <https://allenai.org/privacy-policy> |
+| [Springer Nature](https://dev.springernature.com/)                                                              | <https://dev.springernature.com/privacypolicy> |
+| [The SAO/NASA Astrophysics Data System](https://ui.adsabs.harvard.edu/)                                         | <https://ui.adsabs.harvard.edu/help/privacy/> |
+| [Unpaywall](https://unpaywall.org/)                                                                             | <https://unpaywall.org/legal/privacy> |
+| [zbMATH Open](https://www.zbmath.org)                                                                           | <https://zbmath.org/privacy-policy/> |
+
+&ast; *Note: The Mr. DLib service is used for the related articles tab in the entry editor and collects also your language, your browser and operating system (by default *disabled*).* 
+
+## JabRef Browser Extension
+
+No personal data (like name, email address, billing address or credit card) is collected.
+By using the extension you will remain completely anonymous.
+
+### Collecting information
+
+When you access a website with your web browser, the Browser Extension is designed to save time by allowing you to quickly import bibliographic information into JabRef.
+Therefor the extension collects and analyzes some information concerning your browser activity.
+In particular, the following information is used:
+- the url of the current website you are visiting,
+- the content of the current website.
+
+### Storing information
+No data will be stored by the Browser Extension.
+
+### Sharing information
+The Browser Extension does not share any data except with the local instance of the JabRef software application, which stores the citation data as a new entry in its library.
+
+## Links to other Websites
+- Our Service may contain links to other websites that are not operated by Us. If You click on a third party link, You will be directed to that third party's site. We strongly advise You to review the Privacy Policy of every site You visit.
+- We have no control over and assume no responsibility for the content, privacy policies or practices of any third party sites or services.
+
+## Changes to this Privacy Policy
+This privacy policy may be changed eventually. 
+We encourage you to check this Privacy Policy periodically for any changes.
+Any material change will be mentioned in the changelog and in our blog.
+
+This privacy policy is in effect as of the day mentioned as "last updated" above and will remain in effect except with respect to any changes in its provisions in the future, which will be in effect immediately after being posted on this page.
+
+## Contact
+For any questions or concerns regarding the privacy policy, please send us an email to <vorstand@jabref.org> or write to
+
+JabRef e.V.<br>
```

Comment:
<br> in markdown?

Reply:
https://www.markdownguide.org/basic-syntax#line-break-best-practices